packages/powersync/README.md

```diff
@@ -11,48 +11,141 @@
 * No need for client-side database migrations - these are handled automatically.
 * Subscribe to queries for live updates.
 
+## Examples
+
+For complete app examples, see our [example app gallery](https://docs.powersync.com/resources/demo-apps-example-projects#flutter)
+
+For examples of some common patterns, see our [example snippets](./example/README.md)
+
 ## Getting started
 
+You'll need to create a PowerSync account and set up a PowerSync instance. You can do this at [https://www.powersync.com/](https://www.powersync.com/).
+
+### Install the package
+
+`flutter pub add powersync`
+
+### Implement a backend connector and initialize the Powersync database
+
 ```dart
 import 'package:powersync/powersync.dart';
 import 'package:path_provider/path_provider.dart';
 import 'package:path/path.dart';
 
+// Define the schema for the local SQLite database.
+// You can automatically generate this schema based on your sync rules:
+// In the Powersync dashboard, right-click on your PowerSync instance and then click "Generate client-side schema"
 const schema = Schema([
   Table('customers', [Column.text('name'), Column.text('email')])
 ]);
 
 late PowerSyncDatabase db;
 
-// Setup connector to backend if you would like to sync data.
-class BackendConnector extends PowerSyncBackendConnector {
+// You must implement a backend connector to define how Powersync communicates with your backend.
```

Comment:
```suggestion
// You must implement a backend connector to define how PowerSync communicates with your backend.
```

Reply:
@benitav I'm going to correct all the examples of that in my changes, but note that there are a number of others sprinkled throughout the codebase.

Reply:
Sounds good - I just happened to spot this one :)